src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -52,9 +52,11 @@
     <PackageReference Include="MicroBuild.Core" Version="0.3.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <!--
     <PackageReference Include="PublicApiAnalyzer" Version="1.0.0-alpha001">
```

Comment:
Will enable it back once other comments are addressed. I chose to modify public.shipped.txt only after everything else is ready

Reply:
I start review by looking at these files. Especially when there are so many files in PR. I'd appreciate if you can keep updating API files

Reply:
Understood. I have enabled it back and modified all public api files to reflect correct APIs. ( I disabled it because it was a disturbance while i was still tweaking API!)